### DISCUSSION TITLE
How to set machine.json? I need some help, thank you.

### DISCUSSION THREAD

**Discussion by 13yong** [deepmodeling/dpgen#627] [Q&A]
As a beginner in using DPGEN, I have tried to set this since two days. But it always raise error like this:
![image](https://user-images.githubusercontent.com/90954282/147848753-16207215-4d86-4e51-b7ba-eafa642c552e.png)
My machine.jsion is as follows:
![image](https://user-images.githubusercontent.com/90954282/147848798-f994d2b4-7670-483d-9e7b-7415e0d6672f.png)
![image](https://user-images.githubusercontent.com/90954282/147848810-e3d11acb-b141-4083-b73f-f0e26e58cc03.png)
![image](https://user-images.githubusercontent.com/90954282/147848836-66a40e73-ba8f-4f51-9983-93ac9017886c.png)

My Script is : dpgen init_bulk param.json machine.json
It is note that when I use "dpgen init_bulk param.json" , the running is ok. In this part the param.json only include stage 1 and 2.

**Comment by AnguseZhang** [comment]
What's the reference you followed? The format should be "train" : [{"command" : {} , "resources" : {}}].  
See https://github.com/deepmodeling/dpgen/blob/master/examples/machine/DeePMD-kit-1.x/machine-slurm-qe.json

**Reply by 13yong** [reply]
Firstly, thank you for your answer my question. Under your guidance, my task can run successfully! Thank you so much.
Secondly, I have some questions about dpgen/autotest. I can run autotest like Tongqi Wen‘ video， can not run like our website. There are make, run, post processes. And I also want to ask that autotest can do phonon job? 
In the end, thank you again for your help and wish you success in your work and family harmony.
